packages/stream_video_flutter/lib/src/call_screen/lobby_video.dart

```diff
@@ -112,83 +112,85 @@ class _StreamLobbyVideoState extends State<StreamLobbyVideo> {
     final cameraEnabled = _cameraTrack != null;
     final microphoneEnabled = _microphoneTrack != null;
 
-    return Column(
-      children: [
-        Container(
-          constraints: const BoxConstraints(maxWidth: 420),
-          height: 280,
-          child: ClipRRect(
-            borderRadius: BorderRadius.circular(16),
-            child: DecoratedBox(
-              decoration: BoxDecoration(color: cardBackgroundColor),
-              child: Builder(
-                builder: (context) {
-                  Widget placeHolderBuilder(BuildContext context) {
-                    return Center(
-                      child: StreamUserAvatarTheme(
-                        data: userAvatarTheme,
-                        child: StreamUserAvatar(
-                          user: currentUser,
+    return ConstrainedBox(
+      constraints: const BoxConstraints(maxWidth: 420),
```

Comment:
❔ should we also use the `kMaxWidthRegularScreen` here?

Reply:
Then i would suggest to not make these changes in the SDK and move it to dogfooding.